libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -162,7 +162,7 @@ void AP_Mount_Backend::send_gimbal_manager_information(mavlink_channel_t chan)
     mavlink_msg_gimbal_manager_information_send(chan,
                                                 AP_HAL::millis(),                       // autopilot system time
                                                 get_gimbal_manager_capability_flags(),  // bitmap of gimbal manager capability flags
-                                                _instance,                              // gimbal device id
+                                                _instance + 1,                          // gimbal device id
```

Comment:
can you check with David Sastre on this change? maybe add comment to xml if this is right

Reply:
[We've discussed in this discussion](https://github.com/mavlink/mavlink/pull/1980) and we've agreed it should be 1 based.